curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98fb.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-51
 
 # --description--
 
-Gradient transitions often gradually change from one color to another. You can make the change a solid line like this:
+Gradient transitions often gradually change from one color to another. You can make the gradual change become a solid line like this:
```

Comment:
```suggestion
Gradient transitions often gradually change from one color to another. You can make the transition with a hard stop like this:
```
Maybe something like this? _Hard transition_ or _hard stop_ seem to be the wording used to describe the result.

Reply:
That is good, but what do you think of this:
Gradient transitions often gradually change from one color to another. When a more abrupt change is required, the transition can be made with a "hard stop" like this: